runner/store.py

```diff
@@ -30,4 +30,9 @@ def set_stats(self, id: str, stats: pd.Series) -> None:
         self.stats[id] = stats
 
     def get_all_stats(self) -> pd.DataFrame:
-        return pd.concat(list(self.stats.values()), axis=1)
+        df = pd.DataFrame([])
```

Comment:
isn't this equivalent to `DataFrame.from_dict(self.stats)`?

Reply:
No, unfortunately that only works if it's a simple dict. We have a dict of pd.Series, hence I have to do it this way to also preserve column names.